cover_agent/AICaller.py

```diff
@@ -23,7 +25,7 @@ def call_model(self, prompt, max_tokens=4096):
             tuple: A tuple containing the response generated by the language model, the number of tokens used from the prompt, and the total number of tokens in the response.
         """
         messages = [{"role": "user", "content": prompt}]
-        response = litellm.completion(model=self.model, messages=messages, max_tokens=max_tokens, stream=True)
+        response = litellm.completion(model=self.model, api_base=self.api_base, messages=messages, max_tokens=max_tokens, stream=True) if "ollama" in self.model or "huggingface" in self.model else litellm.completion(model=self.model, messages=messages, max_tokens=max_tokens, stream=True)
```

Comment:
While this is definitely _Pythonic_ I feel like this calls runs a bit too long. Any chance we can break this apart into a multi-line `if` statement for readability purposes?